projects/tuist/fixtures/app_with_spm_dependencies/App/Project.swift

```diff
@@ -39,6 +39,7 @@ let project = Project(
                 .external(name: "IterableSDK"),
                 .external(name: "Stripe"),
                 .external(name: "TYStatusBarView"),
+                .external(name: "GRDB"),
```

Comment:
Can you try to also actually refer its symbols in the code, as done for the other libraries?

Reply:
done